stream-chat-android-core/src/main/java/io/getstream/chat/android/models/Thread.kt

```diff
@@ -21,19 +21,21 @@ import java.util.Date
 
 @Immutable
 public data class Thread(
+    val activeParticipantCount: Int?,
     val cid: String,
-    val channelInfo: ChannelInfo,
+    val channel: Channel?,
     val parentMessageId: String,
-    val parentMessage: Message,
+    val parentMessage: Message?,
```

Comment:
Here I followed the swagger api spec where the `parent_message` field was not marked as mandatory. But you are right, this will probably never be null, I will update this.